src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <>
+    <h3>{title}</h3>
+    <h4>{author}</h4>
+  </>
+);
+
```

Comment:
- [ ] **Good job so far 👏 👏** However, It would be better to add a **button** for removing books even if the **button** wouldn't remove the books yet, this is just a placeholder for now, Kindly try to add this button as it will be useful in the following project 🙏